doc/source/raysgd/raysgd_pytorch.rst

```diff
@@ -115,30 +118,120 @@ You can also set the number of workers and whether the workers will use GPUs:
+            """
+            self.device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+        def train_batch(self, batch, batch_info):
+            """Trains on one batch of data from the data creator.
+
+            Args:
+                batch: One item of the validation iterator.
+                batch_info (dict): Information dict passed in from ``train_epoch``.
+
+            Returns:
+                A dict of metrics. Defaults to "loss" and "num_samples",
+                    corresponding to the total number of datapoints in the batch.
+            """
+            real_label = 1
+            fake_label = 0
+            discriminator, generator = self.models
+            optimD, optimG = self.optimizers
+
+            discriminator.zero_grad()
+            real_cpu = batch[0].to(self.device)
+            batch_size = real_cpu.size(0)
+            label = torch.full((batch_size, ), real_label, device=self.device)
+            output = discriminator(real_cpu).view(-1)
+            errD_real = self.criterion(output, label)
+            errD_real.backward()
+
+            noise = torch.randn(
+                batch_size, generator.latent_vector_size, 1, 1, device=self.device)
+            fake = generator(noise)
+            label.fill_(fake_label)
+            output = discriminator(fake.detach()).view(-1)
+            errD_fake = self.criterion(output, label)
+            errD_fake.backward()
+            errD = errD_real + errD_fake
+            optimD.step()
+
+            generator.zero_grad()
+            label.fill_(real_label)
+            output = discriminator(fake).view(-1)
+            errG = self.criterion(output, label)
+            errG.backward()
+            optimG.step()
+
+            is_score, is_std = self.inception_score(fake)
 
-You can customize the exact function that is called by using a customized training function (see :ref:`raysgd-custom-training`).
+            return {
+                "loss_g": errG.item(),
+                "loss_d": errD.item(),
+                "inception": is_score,
+                "num_samples": batch_size
+            }
 
+    trainer = PyTorchTrainer(
+            model_creator,
+            data_creator,
+            optimizer_creator,
+            nn.BCELoss,
+            training_operator_cls=GANOperator,
+            num_replicas=num_replicas,
+            config=config,
+            use_gpu=True,
+            batch_size=128)
+
+    for i in range(5):
+        stats = trainer.train()
+        print(stats)
+
+See the `DCGAN example <https://github.com/ray-project/ray/blob/master/python/ray/util/sgd/pytorch/examples/dcgan.py>`__ for an end to end example. It constructs two models and two optimizers and uses a custom training operator to provide a non-standard training loop.
+
+
+Training/Validation
+-------------------
+
+Now that the trainer is constructed, you'll naturally want to train the model.
```

Comment:
```suggestion
Now that the trainer is constructed, here's how to train the model.
```
"naturally" sounds like fluff, always good to use simple language where possible for things like these IMO